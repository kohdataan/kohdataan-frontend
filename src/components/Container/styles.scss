.main-container {
  height: 100%;
}

@media screen and (min-width: 960px) {
  .main-container {
    width: 960px;
    margin: 0 auto;
    position: relative;
    height: 100%;
  }
}

.registration-container {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  height: 100%;
}
