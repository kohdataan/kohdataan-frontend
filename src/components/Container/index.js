import React from 'react'
import PropTypes from 'prop-types'
import './styles.scss'

const Container = props => {
  const { className, children } = props
  return <div className={className}>{children}</div>
}

Container.propTypes = {
  className: PropTypes.string.isRequired,
  children: PropTypes.element.isRequired,
}

export default Container
