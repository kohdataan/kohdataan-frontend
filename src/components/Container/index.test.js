import React from 'react'
import { create } from 'react-test-renderer'
import Container from './index'

describe('Container', () => {
  test('it matches the snapshot', () => {
    const component = create(
      <Container className="test">
        <p>Test</p>
        <p>Test</p>
      </Container>
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
