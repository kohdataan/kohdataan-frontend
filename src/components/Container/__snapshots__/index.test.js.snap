// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container it matches the snapshot 1`] = `
<div
  className="test"
>
  <p>
    Test
  </p>
  <p>
    Test
  </p>
</div>
`;
